### ISSUE TITLE
curated datasets

### ISSUE THREAD

**Issue by lila167** [Murali-group/Beeline#52]
Hi, 

Thank you very much for this tool!!
I am wondering if one can generate the curated datasets (Fig. 3 of your paper) with less noise.

Best

**Comment by tmmurali** [comment]
Thank you for the feedback @lila167.

@adyprat is there a way to control the noise from a command-line parameter? I couldn't see one after a quick glance.

**Comment by amoghpj** [comment]
The noise is currently controlled by a hard-coded value in this function https://github.com/Murali-group/BoolODE/blob/b4a752a5ca5087f65d6a58802785b4590c47e29e/BoolODE/simulator.py#L3 
@lila167  we chose to hard-code a nominal value for the strength of noise because larger values did not always yield the expected number of steady states. We have not systematically tried varying the noise parameters to lower values.

**Comment by tmmurali** [comment]
Thanks for explaining, @amoghpj. Is it worthwhile to make it a command line parameter and add your warnings to the help message that goes with the parameter definition?

@lila167 you can of course change the hardcoded value to whatever you desire.

**Comment by lila167** [comment]
Thank you both!! I will give it a try with lower noise level.

**Comment by lila167** [comment]
One more question, is there any way to assess a method using "AUPRC ratio" without integrating it into your pipeline?
@amoghpj If you could tell me where in your codes you measure "AUPRC ratio", would be really helpful. 
Many thanks.

**Comment by amoghpj** [comment]
> Thanks for explaining, @amoghpj. Is it worthwhile to make it a command line parameter and add your warnings to the help message that goes with the parameter definition?
 
Yes, I think it is a good idea to expose this parameter. I will create a pull request with the additions.

**Comment by amoghpj** [comment]
I've created a pull request here https://github.com/Murali-group/BoolODE/pull/17/commits/e8835f9bb104f3bf539ab467ad8703371a67a920 . It would be great if someone can test the patch and confirm that the expected behavior.
